t/typescript/typescript_3.9.9_ubi_8.3.sh

```diff
@@ -0,0 +1,62 @@
+#-----------------------------------------------------------------------------
+#
+# Package	:typescript
+# Version	: 3.9.9
+# Source repo	: https://github.com/Microsoft/TypeScript
+# Tested on	: ubi 8.3
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Adilhusain Shaikh <Adilhusain.Shaikh@ibm.com>
+#
```

Comment:
@adilhusain-s We have recently added some additional fields here like Language and Travis-Check, please update/add those, refer to the latest templates at https://github.com/ppc64le/build-scripts/tree/master/templates